test/cpp/jit/test_gpu.cpp

```diff
@@ -12687,6 +12724,66 @@ TEST(NVFuserTest, FusionSegmentReducePointwise_CUDA) {
       executor_cache.fusion(), outputs, {t0, t1, t2}, {t6}, __LINE__, __FILE__);
 }
 
+TEST(NVFuserTest, FusionMultipleVectorize_CUDA) {
```

Comment:
Need to check these as we shouldn't vectorize until we have a full wave. Should make sure that point is checked correctly, or just make sure we're way in the point we should vectorize on any hardware we actually run on.